docs/configure

```diff
@@ -714,32 +707,13 @@ virtual_domains:
                 and users.domain_id = domains.domain_id}{$value}fail}
   allow_fail
   data = ${extract{smtp}{$address_data}}
+  headers_add = X-Spam-Score: $acl_m_spam_score
+  headers_add = X-Spam-Score-Int: $spam_score_int
+  headers_add = X-Spam-Bar: $acl_m_spam_bar
```

Comment:
If you forward e-mail to your account, these headers will be added twice. Is it possible to put in a condition to add the header only if there isn't such a header already?